apps/src/javalab/Javalab.js

```diff
@@ -313,7 +313,7 @@ Javalab.prototype.init = function (config) {
 
   let backpackApi = null;
   if (backpackEnabled) {
-    backpackApi = new BackpackClientApi(config.backpackChannel);
+    backpackApi = new BackpackClientApi('javalab', config.backpackChannel);
```

Comment:
I think we also need to update [levels_helper](https://github.com/code-dot-org/code-dot-org/blob/52e30bb43a951d3b5742235551fefb4ded674cfc/dashboard/app/helpers/levels_helper.rb#L263-L267) to populate the channel correctly.